.config/dotnet-tools.json

```diff
@@ -3,10 +3,16 @@
   "isRoot": true,
   "tools": {
     "dotnet-serve": {
-      "version": "1.7.125",
+      "version": "1.8.15",
       "commands": [
         "dotnet-serve"
       ]
+    },
+    "dotnet-format": {
+      "version": "5.0.210401",
+      "commands": [
+        "dotnet-format"
```

Comment:
Would we add some spice to our commit hooks by changing this string to `format`?

Reply:
Please clarify your question @pranavkm